src/app/public-view/components/content/components/content-individual-project-photos/content-individual-project-photos.component.ts

```diff
@@ -22,7 +22,8 @@ export class ContentIndividualProjectPhotosComponent implements OnInit {
   private createPhotoFileObservables(individualProjects: Project[]): Observable<SafeUrl>[] {
     const photoFileObservables = [];
     individualProjects.forEach((value: Project) => {
-      photoFileObservables.push(this.fileService.getFileFromPathAsSafeUrl(value.titleImage));
+      // todo
+      // photoFileObservables.push(this.fileService.getFileFromPathAsSafeUrl(value.titleImagePath)); todo fix
```

Comment:
toto by sme tiez v ramci tohto PR mali poriesit

Reply:
V dalsom pr (HFA-135) je to poriesene.